src/phalanx/cli.py

```diff
@@ -143,6 +144,14 @@ def application_add_helm_repos(
         " letter and, with the application name, be less than 80 characters."
     ),
 )
+@click.option(
+    "-p",
+    "--project",
+    type=click.Choice([p.value for p in Project]),
+    prompt="Argo CD project",
+    show_choices=False,
```

Comment:
Mostly it just looked ugly because it makes the prompt much longer than 80 columns, but I don't really like either of the options here. If it would list all the options one per line, that would be much better. Maybe I should just live with the long and kind of ugly prompt? I don't have a strong feeling about this.

Reply:
Yeah, it's tough. From a UX point of view I think people will want to know what their options are, and that weights heavier than strict neatness (though it'd be nice is Click did better formatting). It might be easier from an interactive prompt rather than from a command?

Reply:
I hit on the idea of overriding the prompt string itself to include the options, which seems to basically work.